linkage/demo_notebooks/2022_10_21a_prl_splink_test_part_1.ipynb

```diff
@@ -0,0 +1,3242 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    #         ' and substr(l.first_name, 1,1) = substr(r.first_name, 1,1)'
</pre><p>Why are these parts commented out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>